pysrc/SBBox.cpp

```diff
@@ -35,36 +35,40 @@ namespace galsim {
     struct PySBBox 
```

Comment:
Because I put both of the wrap calls in `pyExportSBBox` rather than make another function to call.  The same way `pyExportSBSersic` also includes the wrap call for `SBDevaucouleurs`.